screens/Tutorials/TutorialScreen1.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import { Image, StyleSheet, TouchableHighlight, View } from "react-native";
+import KitText from "../../components/KitText";
+import Colors from "../../constants/Colors";
+import KitButtonSupreme from "../../components/KitButtonSupreme";
+import CompletionBar from "../../components/challenges/CompletionBar";
+
+export default function Tutorial1Screen(props) {
+  const { navigation } = props;
+  const kitglobeImage = require("../../assets/images/kitglobe.png");
+
+  return (
+    <View style={styles.container}>
+      <TouchableHighlight 
+      onPress = {() => {navigation.navigate("Tutorial2")}}
+      underlayColor='none'
```

Comment:
expo doesn't like the syntax here: 
![image](https://user-images.githubusercontent.com/23728476/70364764-c3328e80-1842-11ea-9aa6-084e224fdb9c.png)

I don't think `'none'` is a valid prop to pass to `underlayColor`. Try one of the supported formats. If you don't want any underlying color try using 
`underlayColor={"transparent"}`